features/test/gezaghebbende-zonder-burgerservicenummer-test.feature

```diff
@@ -0,0 +1,234 @@
+      Dan is het gezag over 'Minderjarige' eenhoofdig ouderlijk gezag met ouder 'Moeder'
+
+    Voorbeeld: twee ouders, vader is ongehuwd, moeder staat niet ingeschreven in de BRP en vader heeft erkend voor 1-1-2023
+      Gegeven de persoon 'Moeder' zonder burgerservicenummer
+      * is meerderjarig
+      * is een vrouw
+      En de persoon 'Vader' met burgerservicenummer '000000024'
+      * is meerderjarig
+      * is een man
+      En de persoon 'Minderjarige' met burgerservicenummer '000000036'
+      * is op 14-10-2022 geboren
+      * is ingeschreven in een Nederlandse gemeente
+      * heeft 'Moeder' en 'Vader' als ouders
+      Als 'gezag' wordt gevraagd van 'Minderjarige'
+      Dan is het gezag over 'Minderjarige' eenhoofdig ouderlijk gezag met ouder 'Moeder'
+
+    Voorbeeld: vader staat niet ingeschreven in de BRP en heeft erkend na 1-1-2023
+      Gegeven de persoon 'Moeder' met burgerservicenummer '000000012'
+      * is meerderjarig
+      * is een vrouw
+      En de persoon 'Vader' zonder burgerservicenummer
+      * is meerderjarig
+      * is een man
+      En de persoon 'Minderjarige' met burgerservicenummer '000000036'
+      * is op 2-3-2023 geboren
+      * is ingeschreven in een Nederlandse gemeente
+      * heeft 'Moeder' en 'Vader' als ouders
+      Als 'gezag' wordt gevraagd van 'Minderjarige'
+      Dan is het gezag over 'Minderjarige' gezamenlijk ouderlijk gezag met ouder 'Moeder' en ouder 'Vader'
+
+    Voorbeeld: moeder staat niet ingeschreven in de BRP en vader heeft erkend na 1-1-2023
+      Gegeven de persoon 'Moeder' zonder burgerservicenummer
+      * is meerderjarig
+      * is een vrouw
+      En de persoon 'Vader' met burgerservicenummer '000000024'
+      * is meerderjarig
+      * is een man
+      En de persoon 'Minderjarige' met burgerservicenummer '000000036'
+      * is op 14-10-2022 geboren
+      * is ingeschreven in een Nederlandse gemeente
+      * heeft 'Moeder' en 'Vader' als ouders
+      Als 'gezag' wordt gevraagd van 'Minderjarige'
+      Dan is het gezag over 'Minderjarige' gezamenlijk ouderlijk gezag met ouder 'Moeder' en ouder 'Vader'
+
+    Voorbeeld: moeder staat niet ingeschreven in de BRP en vader heeft geadopteerd
+      Gegeven de persoon 'Moeder' zonder burgerservicenummer
+      * is meerderjarig
+      * is een vrouw
+      En de persoon 'Adoptievader' met burgerservicenummer '000000024'
+      * is meerderjarig
+      * is een man
+      En de persoon 'Minderjarige' met burgerservicenummer '000000036'
+      * is 15 jaar geleden geboren
+      * is ingeschreven in een Nederlandse gemeente
+      * heeft 'Moeder' als ouder
+      En 'Minderjarige' is morgen 14 jaar geleden geadopteerd door 'Adoptievader'
+      Als 'gezag' wordt gevraagd van 'Minderjarige'
+      Dan is het gezag over 'Minderjarige' gezamenlijk ouderlijk gezag met ouder 'Moeder' en ouder 'Adoptievader'
+
+    Voorbeeld: twee ouders en in gerechtelijke uitspraak is het gezag toegewezen aan beide ouders
```

Comment:
hier wordt één uitspraak in de scenario titel gezet, ipv een placeholder. Kan dat? Deze titel geldt toch alleen voor de eerste regel in de lijst met voorbeelden hieronder?

Reply:
Dat kan inderdaad. Dan wordt het:

```suggestion
    Voorbeeld: twee ouders en in gerechtelijke uitspraak is het gezag toegewezen aan <gerechtelijke uitspraak>
```